_irc/index.md

```diff
@@ -5,12 +5,273 @@ excerpt: IRCv3 Specifications
 index: true
 ---
 
-{% for spec in site.irc %}
-{% if spec.index != true %}
+Right now the IRCv3 specification is distributed as a series of extension
+specifications to IRC protocol version 2.7, also known as
+[RFC1459](http://tools.ietf.org/html/rfc1459). To understand the basis of the
+IRC version 3 protocol, please read RFC1459 followed by the extension
+specifications.
 
-## {{spec.title}}
+IRCv3 is based on a set of specifications. Initially these specifications were
+released as a versioned distribution – these days, however, each specification
+is released when it is stable and has been widely tested.
 
-{{spec.content}}
+Errata updates may be submitted for our specifications. To do so, simply see
+our [contribution](https://github.com/ircv3/ircv3-specifications/blob/master/CONTRIBUTING.md)
+document.
 
-{% endif %}
-{% endfor %}
+
+---
+
+
+## Capability Negotiation
+
+Capability negotiation is a vital part of IRCv3. Capabilities let us implement
+protocol changes in backwards-compatible ways, as well as convey various
+information on joining a server.
+
+This feature is considered essential and all software implementing IRCv3
+**must** implement at least the v3.1 and v3.2 specifications.
+
+The [v3.1 Capability Negotiation spec]({{site.baseurl}}/specs/core/capability-negotiation-3.1.html)
+conveys the basic listing and requesting of capabilities, and lays the
+framework which most IRCv3 specs use.
+
+The [v3.2 Capability Negotiation spec]({{site.baseurl}}/specs/core/capability-negotiation-3.2.html)
+clarifies how to list large numbers of capabilities (which is needed as we add
+more useful features to IRCv3!). It also allows capabilities to be listed with
+values, improving the implementation of features like SASL.
+
+The [`cap-notify` spec]({{site.baseurl}}/specs/extensions/cap-notify-3.2.html)
+allows clients to be sent notifications when caps are added to or removed from
+the server. This is useful in cases like SASL when the authentication layer
+disconnects (and thus, SASL authentication is no longer possible). This
+extension is **automatically enabled** if clients request v3.2 capability
+negotiation.
+
+
+## [Message Tags]({{site.baseurl}}/specs/core/message-tags-3.2.html)
+
+Message tags extend the core framing used with IRC messages, and allow extended
+data to be send with messages. Tags are used by a number of extensions, and we
+consider them a core part of IRCv3 and of supporting new extensions.
+
+This feature is considered essential and all software implementing IRCv3
+**must** implement at least the core Message Tags specification.
```

Comment:
Don't think we should say this. The whole point of dropping the core/extension and version distinctions is that you should be able to implement IRCv3 as you choose. I don't think implementations that don't support any of the tag specs should be considered non-compliant.